flexmeasures/api/v3_0/assets.py

```diff
@@ -7,13 +7,13 @@
 from flask_login import current_user
 from flask_security import auth_required
 from flask_json import as_json
-from flask_sqlalchemy.pagination import SelectPagination
 
 from marshmallow import fields
 import marshmallow.validate as validate
 
 from webargs.flaskparser import use_kwargs, use_args
 from sqlalchemy import select, delete, func
+from flask_sqlalchemy.pagination import SelectPagination
```

Comment:
Hm, the import grouping is weird here, there should be three groups:  
- standard library
- dependencies
- project imports

Do you know why this moved?

Reply:
I think I was the one who even put that there,  I will put it with the other flask related import above